components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -111,7 +158,10 @@ void Nudging::time_interpolation (const int time_s) {
   double time_step_file_d = time_step_file;
   double w_bef = ((time_index+1)*time_step_file-time_s) / time_step_file_d;
   double w_aft = (time_s-(time_index)*time_step_file) / time_step_file_d;
-  
+  std::cout<<"w_bef: "<<w_bef<<std::endl;
```

Comment:
If this is output you needed while debugging, please remove it. Otherwise, use the `log` method.

Reply:
removed